llvm/test/CMakeLists.txt

```diff
@@ -239,6 +239,67 @@ add_lit_testsuite(check-llvm "Running the LLVM regression tests"
   )
 set_target_properties(check-llvm PROPERTIES FOLDER "Tests")
 
+add_lit_testsuite(check-zksync-opt "Running CodeGen LLVM regression tests with ZKsync opt options set"
```

Comment:
Hey @akiramenai !

Could you, please, take a look at this part? This part introduces `make check-zksync-opt` and `make check-zksync-llc` targets in addition to `llvm-check`. Are you good with the naming conventions I've chosen? ([CPR-1662](https://linear.app/matterlabs/issue/CPR-1662/add-simple-ninja-targets-for-opt-and-llc-regression-tests))

Reply:
I suggest a single target and `make check-verify-llvm-codegen` or something. 
Maybe it's better to also add `make check-verify-llvm` that handles all our regression tests because the original idea was to make it easy to perform CI checks locally.

I prefer to evade from discussion should we name zksync/syncvm/era/eravm and not to mention it at all. Moreover we have EVM target as well.

Reply:
Makes sense, yeah. I'll update.